actions/views/root.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {getClientConfig, getLicenseConfig} from 'mattermost-redux/actions/general';
+import * as UserActions from 'mattermost-redux/actions/users';
+
+export function loadMeAndConfig() {
+    return (dispatch) => {
+        const promises = [
+            dispatch(getClientConfig()),
+            dispatch(getLicenseConfig()),
+        ];
+
+        if (document.cookie.indexOf('MMUSERID=') > -1) {
```

Comment:
we should use `if(document.cookie.includes('MMUSERID=') {`

Reply:
Oh yeah, I'll change that. I always forget arrays have that since strings don't

Reply:
Yeah strings have it too